build_scripts/README.md

```diff
@@ -1,21 +1,362 @@
+```
+
+The last variable, LOGSTASH_API_SECRET is used to capture additional values that are not explicitly codified.
+```diff
+! Due to a current issue you have to set all with dummy values else script breaks. 
+```
+Explaining these below.
+
+```json
+{
+  "azure_audit_conn" : "ENDPOINT FOR AZURE AUDIT",
+  "azure_operational_conn" : "ENDPOINT FOR AZURE OPERATIONAL",
+  "azure_signin_conn" : "ENDPOINT FOR AZURE SIGNIN",
+  "azure_o365_conn" :"ENDPOINT FOR AZURE O365",
+  "azure_tcs_security_conn" : "ENDPOINT FOR AZURE TCS",
+  "azure_o365_dlp_conn" : "ENDPOINT FOR AZURE DLP",
+  "azure_atp_conn" : "ENDPOINT FOR AZURE ATP",
+  "azure_audit_consumer" : "CONSUMER NAME FOR AZURE AUDIT",
+  "azure_operational_consumer" : "CONSUMER NAME FOR AZURE OPERATIONAL",
+  "azure_signin_consumer" : "CONSUMER NAME FOR AZURE SIGNIN",
+  "azure_o365_consumer" : "CONSUMER NAME FOR AZURE O365",
+  "azure_tcs_security_consumer" : "CONSUMER NAME FOR AZURE TCS",
+  "azure_o365_dlp_consumer" : "CONSUMER NAME FOR DLP",
+  "azure_atp_consumer" : "CONSUMER NAME FOR AZURE ATP",
+  "azure_storage_conn" : "STORAGE CONNECTION FOR AZURE",
+  "memcached_address" : "SERVERS FOR THE MISP ENRICHMENT AND MISP PROCESSOR CONFIG e.g. \"127.0.0.1\",\"127.0.0.2\"",
+  "dns_server" : "SERVERS FOR THE DNS ENRICHMENT e.g. \"127.0.0.1\",\"127.0.0.2\""
+}
+```
+
+## Getting started
+
+These files need to exist in your running instance, otherwise Logstash cannot load the geoip enrichment.
+```
+/mnt/s3fs_geoip/GeoLite2-City.mmdb
+/mnt/s3fs_geoip/GeoLitePrivate2-City.mmdb
+```
+Either remove geoip enrichment file or just touch these files if you are disabling the enrichment from settings.json.
```

Comment:
@KrishnanandSingh what if someone wants to add in the geoIP enrichment too? How would they get access to it?

Reply:
Added link to the geoip plugin doc.